apis/validator/prepare_beacon_proposer.yaml

```diff
@@ -12,7 +12,7 @@ post:
 
     Optionally the validator may also include a signed validator registration
     which will be used by the Builder API to verify a validator's latest fee
-    recipient.
+    recipient and preferred gas target.
```

Comment:
should we specify as gas target or gas limit?

if we specify gas target then software has to worry about the 1559 elasticity multiplier which is another constant builders need to track

if you look at stock geth today, it accepts the gas limit as a "ceiling" parameter rather than the 1559 long-run target and when building a block will target the limit towards this ceiling: https://github.com/ethereum/go-ethereum/blob/master/miner/worker.go#L997

the only place geth seems to work in terms of targets is internal to the base fee calculation

Reply:
update: we went w/ limit